fitness_vr.dm

```diff
@@ -0,0 +1,178 @@
+/obj/structure/fitness/boxing_ropes

```

Comment:
You did something weird here, adding this file to the root of the repo rather than where it should go.